mypy/checkmember.py

```diff
@@ -821,12 +820,18 @@ class A(Generic[T]):
         def foo(cls: Type[Q]) -> Tuple[T, Q]: ...
 
     class B(A[str]): pass
-
     B.foo()
 
-    original_type is the value of the type B in the expression B.foo() or the corresponding
-    component in case if a union (this is used to bind the self-types); original_vars are type
-    variables of the class callable on which the method was accessed.
+    Args:
+        t: Declared type of the method (or property);
```

Comment:
Style nit: We don't use `;` after each description of an argument in docstrings.